lib/swaps/SwapClientManager.ts

```diff
@@ -213,6 +214,9 @@ class SwapClientManager extends EventEmitter {
       initWalletPromises.push(keystorePromise);
     }
 
+    // TODO: pass seed to Connext SwapClient
+    console.log(`ConnextClient.initWallet: ${connextSeed}`);
```

Comment:
This temporary console.log is intentional as I wasn't able to use `_connextSeed` to silence it when using object destructuring.

Reply:
You could just leave it off the type declaration for the parameter in the method signature, right? It can still be passed in when calling the method for now, and then can be added in when it's actually being used.

Not a big deal, though.